tests/urls.py

```diff
@@ -0,0 +1 @@
+urlpatterns = []
```

Comment:
We may just be able to remove the reference from the settings file and not have a URLs file. Test that?